_posts/2017-06-05-thanks-electronconf.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+author: wyatt
+title: Thanks, ElectronConf
+image: 'posts/electronconf.jpg'
+credit: Tina Hartung
+---
+
+Over the weekend, my usual sources for industry news exploded in response to [the postponement](http://electronconf.com/) of the first ElectronConf because the panel of speakers did not "reflect the standards to which we hold ourselves."
+
+First: _thank you_. Thank you to ElectronConf and to GitHub for self reflection and recognizing when something is wrong. We need more leading members of our industry to stand up for diversity. At Apsis Labs, we don't develop for or with Electron, but Noah and myself will be attending ElectronConf (whenever it does take place) to show support for taking an active role in increasing the diversity of our community. I hope others will do the same.
+
+Second, to the people decrying this move as being itself prejudiced, you embarrass me. It is shameful that you have gaul to stand up and say that the way things are is the way things should remain, and that taking affirmative action to increase the diversity of tech is wrong.
```

Comment:
"Second, the response from the segments of the tech community that consider this move itself to be prejudiced embarrasses me".

It feels more natural to be embarrassed by a group we feel associated with than it does to "the people expressing X opinion" to me. I don't personally feel associated with "people that express X opinion", but I _do_ feel embarrassed that those people come from the tech community.